akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/ActorSystemSpec.scala

```diff
@@ -48,10 +48,8 @@ class ActorSystemSpec extends WordSpec with Matchers with BeforeAndAfterAll
             inbox.hasMessages should ===(true)
           }
           inbox.receiveAll() should ===("hello" :: Nil)
+          sys.whenTerminated.futureValue
```

Comment:
I think this might be easier, similar to second test case did. Then simplify the `withSystem` helper.

Reply:
Good